soc/arc/snps_qemu/Kconfig.defconfig

```diff
@@ -9,7 +9,7 @@ config SOC
 
 config SYS_CLOCK_HW_CYCLES_PER_SEC
 	int
-	default 10000000
```

Comment:
50 Mhz is probably ok, but wasn't this the real problem for the 'timeouts'? QEMU configured for 100_000 and Zephyr configured for 1_000_000 cycles/sec?

Reply:
I have tried to fix QEMU configuration and Zephyr configuration both to 10Mhz, but test: tests/kernel/mem_protect/syscalls/
still timeout. execution time:

- 10Mhz: 

```
real    2m1.306s
user    2m1.143s
sys     0m0.459s
```
- 50Mhz:
```
real    0m4.991s
user    0m4.869s
sys     0m0.485s
```

Reply:
Do you understand why this is? With iCount on (and the right shift values), it shouldn't matter that much. Maybe at 1Mhz the tick counter overhead too high?  (e.g. 1000 ticks/sec means only 1000 cycles/instruction for the handler).
Just increasing numbers without understanding why it is needed (and why 50 is  ok, instead of 10 or 100...) worries me a bit.

Reply:
I can't reproduce it now, 10MHz works fine too.

Reply:
ok - please change the PR first message then to not mention changing to 50 MHz, but from 1 to 10 MHz. Any QEMU_ICOUNT_SHIFT to be updated as well?

Reply:
we intend to use 10MHz, so the QEMU_ICOUNT_SHIFT is right.